firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -128,6 +136,11 @@ public InstallationResponse createFirebaseInstallation(
       httpsURLConnection.setRequestMethod("POST");
       httpsURLConnection.setDoOutput(true);
 
+      // Set the iid token header for authenticating the iid migration to FIS.
```

Comment:
Note: Set the iid token header for authenticating the Instance-ID migrating to FIS.

Reply:
Done.